boards/common/nrf52/include/cfg_spi_default.h

```diff
@@ -36,7 +36,9 @@ static const spi_conf_t spi_config[] = {
         .sclk = GPIO_PIN(0, 15),
         .mosi = GPIO_PIN(0, 13),
         .miso = GPIO_PIN(0, 14),
+#ifdef ERRATA_SPI_SINGLE_BYTE_WORKAROUND
         .ppi = 0,
```

Comment:
It's 0 by default, so you can do without the ifdef block.

Reply:
I think I rather have it explicitly configured here to prevent nasty suprises.

Reply:
what would be a possible nasty surprise?

Reply:
Somebody modifying this instantiation to multiple SPI configs for his own board and not noticing that the `.ppi` should be added, and thus having only one functional SPI bus.

Same as that a PPI channel is also explicitly set [here] while that one could also be omitted

[here]: https://github.com/RIOT-OS/RIOT/blob/master/boards/nrf51dk/include/periph_conf.h#L68